Kitodo-DataAccess/src/test/resources/xmlWriterTest-expectedResult.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<mets:mets xmlns:mets="http://www.loc.gov/METS/" xmlns:mods="http://www.loc.gov/mods/v3" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
It is possible to add a license header in this file? If so, please add the license header.

Reply:
No, this file is against what the test compares. It cannot be changed, sorry.

Reply:
If I understood your code correct then comments should be ignored.

Reply:
No. XMLWriterTest uses Apache FileUtils to do a compare if the output of the XML writer is equal to this file: `assertThat(FileUtils.contentEquals(testfile, expectedResult), is(true));`

Comments are ignored in the XML Reader, this is what you might be thinking of, but that’s a different thing.